windows_service/datadog_checks/windows_service/data/conf.yaml.example

```diff
@@ -13,17 +13,29 @@ init_config:
 #
 instances:
 
-    ## @param services - list of strings - required
+    ## @param services - (list of string or mapping) - required
     ## List of services to monitor e.g. Dnscache, wmiApSrv, etc.
     ##
-    ## If any service is set to `ALL`, all services registered with the SCM will be monitored.
+    ## Services can also be selectively monitored by their startup type.
+    ## Possible values for the `startup_type` pattern:
+    ##   - disabled
+    ##   - manual
+    ##   - automatic
+    ##   - automatic_delayed_start
```

Comment:
should we add "trigger" facet?

Reply:
I think we should keep it to the list that is selectable from the service manager UI. The triggers really add a whole other layer of complexity to the service behavior though...

I think it might be better as another tag, like `windows_service_trigger_start:[true,false]` and/or as a separate filter `trigger_start: [true,false]`.

It itches me a little bit to handle delayed start one way and trigger start a different way, but Microsoft's already done that...

what do you think?

Reply:
I like `separate filter trigger_start: [true,false].`. I am not 100% sure if it would have a practical utility. E.g. I would like to watch all services which with trigger start which are not running. But would be interested to see what you and @lrbhatia and @julien-lebot would think.